### PR TITLE
Add a bio file for kraymond

### PR DESCRIPTION
**Pull request by kraymond-githubdotcom**
@kfeudner @kmackiewicz

### CODE REVIEW

File: bios/kraymond-bio.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Not part of the Kayla club.  Really.
```
Comment (PaulaLupu) [inline bios/kraymond-bio.md]: test!

Comment (kmackiewicz) [inline bios/kraymond-bio.md]: You can be a part time Kayla member

Comment (kraymond-githubdotcom) [inline bios/kraymond-bio.md]: @kmackiewicz Cool, you can be a part time Dyn employee.

**Comment by kfeudner** [comment]
The trick to being a part time Kayla is figuring out when you are a Kayla

**Pull request merged**